src/lando/api/tests/test_transplants.py

```diff
@@ -962,6 +962,25 @@ def test_integrated_transplant_records_approvers_peers_and_owners(
     assert approved_by == [[101], [102]]
 
     assert hg_landing_worker.run_job(job)
+    assert job.landed_commit_id
+
+    # Fetch Job data.
+    response = proxy_client.get(
+        f"/landing_jobs/{job.id}",
+    )
+    assert (
+        response.status_code == 200
+    ), f"Invalid status code from GET /landing_jobs/{job.id}"
+    assert (
+        response.json["id"] == job.id
+    ), f"Invalid id in GET /landing_jobs/{job.id} response"
+    assert (
+        response.json["status"] == JobStatus.LANDED
+    ), f"Invalid status in GET /landing_jobs/{job.id} response"
+    assert (
+        response.json["commit_id"] == job.landed_commit_id
+    ), f"Invalid commit_id in GET /landing_jobs/{job.id} response"
```

Comment:
I don't think we should be adding new views that utilize the legacy "proxy" client -- it only adds more tech debt.